src/Services/OrganizationService.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+/**
+ * InsuranceService
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Vishnu Yarmaneni <vardhanvishnu@gmail.com>
```

Comment:
also need a copyright line in your headers

Reply:
Done